native/jest.setup.ts

```diff
@@ -25,7 +25,11 @@ const walkDir = (dir: string, callback: (filePath: string) => void): void => {
   fs.readdirSync(dir).forEach(f => {
     const filePath = path.join(dir, f)
     const isDirectory = fs.statSync(filePath).isDirectory()
-    isDirectory ? walkDir(filePath, callback) : callback(filePath)
+    if (isDirectory) {
```

Comment:
Why not ternery?

Reply:
https://eslint.org/docs/rules/no-unused-expressions
> An unused expression which has no effect on the state of the program indicates a logic error.

> For example, n + 1; is not a syntax error, but it might be a typing mistake where a programmer meant an assignment statement n += 1; instead. Sometimes, such unused expressions may be eliminated by some build tools in production environment, which possibly breaks application logic.

It is possible to disable this for ternaries though.